site/public/js/forum.js

```diff
@@ -34,12 +34,15 @@ function openFileForum(directory, file, path ){
     window.open(url,"_blank","toolbar=no,scrollbars=yes,resizable=yes, width=700, height=600");
 }
 
-function checkForumFileExtensions(files){
-    var count = 0;
+function checkForumFileExtensions(post_box_id, files){
+    var count = files.length;
     for(var i = 0; i < files.length; i++){
         var extension = getFileExtension(files[i].name);
-        if(extension == "gif" || extension == "png" || extension == "jpg" || extension == "jpeg" || extension == "bmp"){
-            count++;
+        if( !(extension == "gif" || extension == "png" || extension == "jpg" || extension == "jpeg" || extension == "bmp") ){
```

Comment:
```suggestion
        if(!['gif', 'png', 'jpg', 'jpeg', 'bmp'].includes(extension)) {
```

Reply:
I've combined all of the changes into a commit :)